plugins/pastefromword/filter/default.js

```diff
@@ -533,7 +519,7 @@
 				return;
 			}
 
-			if ( value === '' ) {
+			if ( value === '' || value == null ) {
```

Comment:
I will remove this change because it's not important for this PR, although I'm curious how could it remove styles with `0` value? In JS `0 == null` and `'0' == null` gives false logical value. I'm using double equal operator here to perform `null == undefined` coercion.